src/components/CouponDetails/index.jsx

```diff
@@ -148,7 +148,7 @@ class CouponDetails extends React.Component {
   getTableFilterVisibilitySelectionOptions() {
     return [{
       label: 'Both',
-      value: 'both',
+      value: undefined,
```

Comment:
I prefer `null` to `undefined` if something is intentionally falsey. (Though with React sometimes there's reasons that doesn't work.)